ipamd/ipamd.go

```diff
@@ -651,9 +656,7 @@ func (c *IPAMContext) tryAssignIPs() (increasedPool bool, err error) {
 	eni := c.dataStore.GetENINeedsIP(c.maxIPsPerENI, c.useCustomNetworking)
 	if eni != nil && len(eni.IPv4Addresses) < c.maxIPsPerENI {
 		currentNumberOfAllocatedIPs := len(eni.IPv4Addresses)
-		log.Debugf("Found ENI %s that has less than the maximum number of IP addresses allocated: cur=%d, max=%d", eni.ID, currentNumberOfAllocatedIPs, c.maxIPsPerENI)
 		// Try to allocate all available IPs for this ENI
-		// TODO: Retry with back-off, trying with half the number of IPs each time
```

Comment:
Is this no longer a TODO? Are we not going to try a backoff (other than "if >1 allocation failed, try a single IP allocation" strategy)?

Reply:
Yes, this is a left over comment from another approach of allocating ENIs and it is not valid any more.